src/mds/CInode.cc

```diff
@@ -4780,6 +4780,8 @@ void CInode::validate_disk_state(CInode::validated_data *results,
                            false);
         // Flag that we repaired this BT so that it won't go into damagetable
         results->backtrace.repaired = true;
+        damage_entry_id_t damage_id = mdcache->mds->damage_table.inode_to_id(in->ino());
```

Comment:
better name instead of `damage_id`? Maybe `damaged_inode_id`

Reply:
Replaced with `damaged_entry_id`

Reply:
I don't fret about variable names, but, damage_id looks fine to me.